app/src/main/java/net/gsantner/opoc/util/GsContextUtils.java

```diff
@@ -128,6 +128,7 @@
 
 import com.google.android.material.snackbar.Snackbar;
 
```

Comment:
Dont import app specific ID here (markor R). If possible make the added thing in code dynamically, or lookup the id by the resources helpers that are here.